app/scripts/modules/core/src/search/global/globalSearch.less

```diff
@@ -1,5 +1,9 @@
 @import (reference) "~core/presentation/less/imports/commonImports.less";
 
+global-search {
```

Comment:
i added this (hopefully) temporarily because when there's an angular template override for `SpinnakerHeader`, `react2angular` wraps `GlobalSearch`'s markup in a `<global-search>` tag, which messes up our selectors for the nav (we use direct descendent selectors).

Reply:
Wishful thinking

Reply:
😆 I wasn't gonna say it